ci/jenkins/utils.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function check_and_upgrade_golang() {
+    echo "====== Checking installed Golang version ======"
+    antrea_golang_version="go$(grep -E "^go\ " go.mod | awk '{print $2}')"
+    golang_minor_version=$(echo ${antrea_golang_version} | awk -F '.' '{print $2}')
```

Comment:
This (`golang_minor_version`) is not used, probably a red flag

Reply:
Removed, thanks for the reminder.